vnext/Desktop/ABI/TestController.h

```diff
@@ -15,6 +15,12 @@ struct TestController {
   static Microsoft::ReactNative::IReactModuleBuilder CreateReactModuleBuilder(
       Microsoft::ReactNative::IReactContext context);
   static Microsoft::ReactNative::IReactPackageBuilder CreateReactPackageBuilder();
+  static Microsoft::ReactNative::IRedBoxErrorFrameInfo
+  CreateRedBoxErrorFrameInfo(hstring file, hstring method, uint32_t line, uint32_t column);
```

Comment:
>CreateRedBoxErrorFrameInfo [](start = 2, length = 26)

weird line formatting..

Reply:
clang format, I'm not happy about this :-/

Reply:
Could we change this format? Seems like we should be using format that we like :)

---
In reply to: [456160280](https://github.com/microsoft/react-native-windows/pull/5544#discussion_r456160280) [](ancestors = 456160280)